plugin/api.go

```diff
@@ -144,6 +144,9 @@ type API interface {
 	// UpdateTeam updates a team.
 	UpdateTeam(team *model.Team) (*model.Team, *model.AppError)
 
+	// SearchTeams search a team.
```

Comment:
Could you please document the minimum server version?

```suggestion
	// SearchTeams search a team.
	//
	// Minimum server version: 5.6
```

Reply:
done @hanzei